src/components/action-list/action-list.scss

```diff
@@ -51,6 +51,12 @@
       align-self: stretch;
     }
 
+    &--equal-width {
+      flex-basis: 0;
```

Comment:
afaikt this has issues in older safari and IE, can you confirm/deny?

Reply:
@vergilius thx, I will check it.

Reply:
@vergilius IE11 and older Safari seem's to be working fine.